### DISCUSSION TITLE
Code Search is not working currently on this repository

### DISCUSSION THREAD

**Discussion by surajsavita** [helmfile/helmfile#154] [Questions] [answered]
Hi 
Currently this repository is forked from previous repository so code search functionality is not working on github as shown in the below image. I have cloned to search through code.
Is there anything we can do to have code search functionality working on this repository.

![image](https://user-images.githubusercontent.com/5236276/173034647-091793c1-fe2a-453d-9ed5-542cd6c1547c.png)

Thank you!

**Comment by philmtd** [comment, marked as answer]
Removing the fork relationship would solve this issue. I created a discussion for that here: #191

**Reply by surajsavita** [reply]
Thanks @philmtd
